discord/gateway.py

```diff
@@ -892,16 +892,16 @@ async def initial_connection(self, data):
         state.voice_port = data["port"]
         state.endpoint_ip = data["ip"]
 
-        packet = bytearray(70)
+        packet = bytearray(74)
         struct.pack_into(">H", packet, 0, 1)  # 1 = Send
         struct.pack_into(">H", packet, 2, 70)  # 70 = Length
         struct.pack_into(">I", packet, 4, state.ssrc)
```

Comment:
The fixes were made by minn on dpy

I'm actually not sure if that should change too.
They did not changed it there, but the 70 confuses me.

I don't have a clue what i'm doing here.

@BobDotCom do you have an idea?